src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -41,20 +41,43 @@ const TutorProfile = () => {
     defaultResponse: defaultResponses.array
   })
 
+  useEffect(() => {
+    if (
+      !loading &&
+      response &&
+      response.videoLink &&
+      response.videoLink.tutor !== null &&
+      response.videoLink.tutor !== ''
```

Comment:
what a valid falsy value are you expecting here, that just `... && response.videoLink.tutortutorVideo `is not enough?

Reply:
this implementation is outdated.
Now the check is implemented differently